chia/wallet/wallet_interested_store.py

```diff
@@ -21,7 +21,7 @@ async def create(cls, wrapper: DBWrapper):
         self.db_connection = wrapper.db
         self.db_wrapper = wrapper
         await self.db_connection.execute("pragma journal_mode=wal")
```

Comment:
These two lines can be removed because it's set it previously in wallet state manager